src/clients/web3/getWeb3NoAccount.ts

```diff
@@ -1,18 +1,15 @@
 import Web3 from 'web3';
-import getNodeUrl from './getRpcUrl';
 
-const RPC_URL = getNodeUrl();
+import { RPC_URL } from 'config';
 
 // @todo: we will get the "No 'Access-Control-Allow-Origin' header is present on the requested resource"
 // error on the frontend in testnet when the wallet is unconnected, because the bsc official testnet endpoints
 // don't support CORS request right now, and we didn't find any working testnet endpoints for
 // HTTPProvider or WebSocketProvider neither.
-const getWeb3NoAccount = () => {
+export default function getWeb3NoAccount() {
```

Comment:
I personally prefer arrow functions unless there is a reason to use the function declaration.
The main benefits I like from arrow functions are they are more concise and the lexical binding of `this` is more intuitive